src/__test__/Rocket.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Rocket from '../components/Rocket';
+
+describe('Rocket component', () => {
+  const rocket = {
+    id: 'falcon1',
+    name: 'Falcon 1',
+    description: 'Small launch vehicle',
+    flickr_images: ['https://www.example.com/falcon1.jpg'],
+    reserved: true,
+  };
+
+  test('renders when rocket is reserved', () => {
+    const tree = renderer
+      .create(<Rocket rocket={rocket} handleReserveClick={() => {}} onCancelClick={() => {}} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('renders correctly when rocket is not reserved', () => {
+    const newRocket = { ...rocket, reserved: false };
+    const tree = renderer
+      .create(<Rocket rocket={newRocket} handleReserveClick={() => {}} onCancelClick={() => {}} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] @team great work on your tests, however, I didn't see tests for some components you have 9 components and I could only see tests for 4 components, also tests for pure functions in your redux files.
![image](https://user-images.githubusercontent.com/61408860/235168580-c15aeccf-da4d-4420-90f5-f4ff8d329422.png)

Reply:
@danmainah dear we have only four stand-alone in the component folder even if you count the files one by one there are only exist 5 js files in the component folder. would you please enlighten me and name those nine components individually ?

Reply:
- [ ] @MMhaidari Yes in the component folder you have four, but also add the ones in the route folder, as you can see in your project description it says all components, I would suggest you ask for a second opinion if you feel you disagree with some of the changes requested